pkg/kv/kvserver/queue.go

```diff
@@ -1175,15 +1175,13 @@ func (bq *baseQueue) finishProcessingReplica(
 	// Handle failures.
 	if err != nil {
 		benign := benignerror.IsBenign(err)
-		storeBenign := benignerror.IsStoreBenign(err)
 
 		// Increment failures metric.
 		//
 		// TODO(tschottdorf): once we start asserting zero failures in tests
 		// (and production), move benign failures into a dedicated category.
 		bq.failures.Inc(1)
-		if storeBenign {
-			bq.storeFailures.Inc(1)
+		if benign {
```

Comment:
This is a behavior change - before your PR, we were never entering the `storeBenign` branch. Now we're introducing a number of errors that do. We don't want to introduce a behavior change here, which means this conditional should be removed.